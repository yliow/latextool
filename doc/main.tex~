\input{myarticlepreamble}
\input{yliow}
\renewcommand\TITLE{latextool}

\usetikzlibrary{shapes.geometric}

\begin{document}
\topmatter


\newpage\myinput{notes.tex}
\newpage\myinput{testmylist.tex}
\newpage\myinput{execute.tex}
\newpage\myinput{console.tex}
\newpage\myinput{python.tex}
\newpage\myinput{verbatim.tex}
\newpage\myinput{textcolor.tex}
\newpage\myinput{table.tex}

\newpage\myinput{plot.tex}
\newpage\myinput{grid.tex}

\newpage\myinput{tabrect.tex}

\newpage\myinput{scaling.tex}
\newpage\myinput{line.tex}
\newpage\myinput{circle.tex}

\newpage\myinput{rect.tex}

\newpage\myinput{array.tex}

\newpage\myinput{rectcontainer.tex}
\newpage\myinput{snipped-array.tex}

\newpage\myinput{singlylinkedlist.tex}

\newpage\myinput{doublylinkedlist.tex}
\newpage\myinput{position.tex}
\newpage\myinput{tree.tex}
\newpage\myinput{tree2.tex}

\newpage\myinput{graph.tex}

\begin{comment}
\newpage\myinput{chunkedarray.tex}
\newpage\myinput{array2d.tex}
\newpage\myinput{bend.tex}
\newpage\myinput{arc.tex}
\newpage\myinput{shell.tex}

\newpage\myinput{frame.tex}
\newpage\myinput{test-verbatim-spacing.tex}

\newpage\myinput{automata.tex}

\newpage\myinput{vec2d.tex}

\newpage\myinput{line-graph.tex}


\newpage
\section{TEST ISOCELES TRIANGLE}


\begin{tikzpicture}[
  inner/.style={circle,draw,minimum width=7mm,inner sep=0},
  leaf/.style={isosceles triangle,draw,shape border rotate=90,isosceles triangle stretches=true, minimum height=20mm,minimum width=12mm,inner sep=0,yshift={-20mm},font=\tiny},
  large leaf/.style={leaf,minimum height=35mm,yshift={-14.5mm}},
  level 1/.style={sibling distance=30mm},
  level 2/.style={sibling distance=21mm},
  level 3/.style={sibling distance=14mm},
]
  \node[inner] {T}
    [child anchor=north]
    child {node[inner] {L}
      child {node[large leaf] {}}
      child {node[inner] {LR}
        child{node[leaf] {}}
        child{node[leaf] {}}}}
    child {node[large leaf] {L.Allison}};
\end{tikzpicture}

\verb!http://tex.stackexchange.com/questions/7862/triangle-node-with-adjustable-height!

\verb!http://tex.stackexchange.com/questions/37462/placing-a-triangle-around-nodes-in-a-tree!

\end{comment}
\end{document}
