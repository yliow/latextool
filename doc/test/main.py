from math import sin
domain = [x / 10.0 for x in range(-100, 101)]
y_sinx = [(x, 5 * sin(x)) for x in domain]
y_x = [(x, x) for x in domain]
from latextool_basic import *
plot = FunctionPlot(width="6in", height="3in")
plot.add(y_sinx, line_width='2', color='red', legend=r'$y = 5 \sin(x)$')
plot.add(y_x, line_width='2', color='green', legend='$y = x$')
print plot

