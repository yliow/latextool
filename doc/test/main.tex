\input{myblankpreamble2}

\begin{document}

\end{document}
