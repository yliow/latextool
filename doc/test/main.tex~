\input{myblankpreamble}
\begin{document}
hello world
\end{document}
