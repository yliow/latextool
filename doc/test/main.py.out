
\begin{center}
\begin{tikzpicture}[line width=1]
\begin{axis}[width=6in, height=3in,
             xlabel={\mbox{}},
             xlabel style={name=xlabel}, 
             ylabel={\mbox{}},
             legend style={
                at={(xlabel.south)},
                yshift=-1ex,
                anchor=north,
                legend cell align=left
            }
        ]
]
\addplot[draw=red, line width=2] coordinates {(-10.0, 2.7201055544468487)
(-9.9, 2.2876794688766067)
(-9.8, 1.8323956462596418)
(-9.7, 1.3588031320547123)
(-9.6, 0.8716339061148982)
(-9.5, 0.3757556023090465)
(-9.4, -0.12387712726678882)
(-9.3, -0.6222721175353085)
(-9.2, -1.1144495705012383)
(-9.1, -1.5954918117467607)
(-9.0, -2.060592426208783)
(-8.9, -2.505104282289423)
(-8.8, -2.9245859644588084)
(-8.7, -3.314846150410917)
(-8.6, -3.6719854893705666)
(-8.5, -3.9924355631174513)
(-8.4, -4.2729945404414025)
(-8.3, -4.510859168781466)
(-8.2, -4.703652783398866)
(-8.1, -4.849449054225431)
(-8.0, -4.946791233116909)
(-7.9, -4.99470670919886)
(-7.8, -4.992716726873025)
(-7.7, -4.940841169385002)
(-7.6, -4.8395983601574315)
(-7.5, -4.6899998838736945)
(-7.4, -4.493540479058135)
(-7.3, -4.252183103142822)
(-7.2, -3.9683393192457657)
(-7.1, -3.64484520062938)
(-7.0, -3.2849329935939453)
(-6.9, -2.8921988219410006)
(-6.8, -2.470566755693041)
(-6.7, -2.0242496030829917)
(-6.6, -1.5577068175668893)
(-6.5, -1.0755999404390777)
(-6.4, -0.5827460242524682)
(-6.3, -0.08406950242174857)
(-6.2, 0.41544701408748197)
(-6.1, 0.9108125213604794)
(-6.0, 1.3970774909946293)
(-5.9, 1.8693833241511801)
(-5.8, 2.323010897068787)
(-5.7, 2.753427712988188)
(-5.6, 3.1563331893616082)
(-5.5, 3.5277016278519597)
(-5.4, 3.8638224377799357)
(-5.3, 4.161337211119506)
(-5.2, 4.417273278600765)
(-5.1, 4.629073411638663)
(-5.0, 4.794621373315692)
(-4.9, 4.912263063121662)
(-4.8, 4.980823044179203)
(-4.7, 4.999616287820504)
(-4.6, 4.968455018167322)
(-4.5, 4.887650588325485)
(-4.4, 4.75801036944758)
(-4.3, 4.580829683747274)
(-4.2, 4.357878862067941)
(-4.1, 4.091385555322051)
(-4.0, 3.7840124765396412)
(-3.9, 3.438830795919869)
(-3.8, 3.0592894547135945)
(-3.7, 2.649180704542467)
(-3.6, 2.2126022164742625)
(-3.5, 1.7539161384480992)
(-3.4, 1.2777055101341561)
(-3.3, 0.7887284707162411)
(-3.2, 0.2918707171379004)
(-3.1, -0.20790331216645247)
(-3.0, -0.7056000402993361)
(-2.9, -1.1962466460699122)
(-2.8, -1.6749407507795255)
(-2.7, -2.136899401169149)
(-2.6, -2.577506859107321)
(-2.5, -2.9923607205197826)
(-2.4, -3.377315902755755)
(-2.3, -3.7285260608836013)
(-2.2, -4.0424820190979505)
(-2.1, -4.316046833244369)
(-2.0, -4.546487134128409)
(-1.9, -4.731500438437072)
(-1.8, -4.869238154390976)
(-1.7, -4.958324052262343)
(-1.6, -4.997868015207525)
(-1.5, -4.987474933020272)
(-1.4, -4.9272486499423005)
(-1.3, -4.817790927085965)
(-1.2, -4.660195429836131)
(-1.1, -4.456036800307177)
(-1.0, -4.207354924039483)
(-0.9, -3.916634548137417)
(-0.8, -3.586780454497614)
(-0.7, -3.221088436188455)
(-0.6, -2.8232123669751767)
(-0.5, -2.397127693021015)
(-0.4, -1.9470917115432527)
(-0.3, -1.4776010333066978)
(-0.2, -0.9933466539753061)
(-0.1, -0.4991670832341408)
(0.0, 0.0)
(0.1, 0.4991670832341408)
(0.2, 0.9933466539753061)
(0.3, 1.4776010333066978)
(0.4, 1.9470917115432527)
(0.5, 2.397127693021015)
(0.6, 2.8232123669751767)
(0.7, 3.221088436188455)
(0.8, 3.586780454497614)
(0.9, 3.916634548137417)
(1.0, 4.207354924039483)
(1.1, 4.456036800307177)
(1.2, 4.660195429836131)
(1.3, 4.817790927085965)
(1.4, 4.9272486499423005)
(1.5, 4.987474933020272)
(1.6, 4.997868015207525)
(1.7, 4.958324052262343)
(1.8, 4.869238154390976)
(1.9, 4.731500438437072)
(2.0, 4.546487134128409)
(2.1, 4.316046833244369)
(2.2, 4.0424820190979505)
(2.3, 3.7285260608836013)
(2.4, 3.377315902755755)
(2.5, 2.9923607205197826)
(2.6, 2.577506859107321)
(2.7, 2.136899401169149)
(2.8, 1.6749407507795255)
(2.9, 1.1962466460699122)
(3.0, 0.7056000402993361)
(3.1, 0.20790331216645247)
(3.2, -0.2918707171379004)
(3.3, -0.7887284707162411)
(3.4, -1.2777055101341561)
(3.5, -1.7539161384480992)
(3.6, -2.2126022164742625)
(3.7, -2.649180704542467)
(3.8, -3.0592894547135945)
(3.9, -3.438830795919869)
(4.0, -3.7840124765396412)
(4.1, -4.091385555322051)
(4.2, -4.357878862067941)
(4.3, -4.580829683747274)
(4.4, -4.75801036944758)
(4.5, -4.887650588325485)
(4.6, -4.968455018167322)
(4.7, -4.999616287820504)
(4.8, -4.980823044179203)
(4.9, -4.912263063121662)
(5.0, -4.794621373315692)
(5.1, -4.629073411638663)
(5.2, -4.417273278600765)
(5.3, -4.161337211119506)
(5.4, -3.8638224377799357)
(5.5, -3.5277016278519597)
(5.6, -3.1563331893616082)
(5.7, -2.753427712988188)
(5.8, -2.323010897068787)
(5.9, -1.8693833241511801)
(6.0, -1.3970774909946293)
(6.1, -0.9108125213604794)
(6.2, -0.41544701408748197)
(6.3, 0.08406950242174857)
(6.4, 0.5827460242524682)
(6.5, 1.0755999404390777)
(6.6, 1.5577068175668893)
(6.7, 2.0242496030829917)
(6.8, 2.470566755693041)
(6.9, 2.8921988219410006)
(7.0, 3.2849329935939453)
(7.1, 3.64484520062938)
(7.2, 3.9683393192457657)
(7.3, 4.252183103142822)
(7.4, 4.493540479058135)
(7.5, 4.6899998838736945)
(7.6, 4.8395983601574315)
(7.7, 4.940841169385002)
(7.8, 4.992716726873025)
(7.9, 4.99470670919886)
(8.0, 4.946791233116909)
(8.1, 4.849449054225431)
(8.2, 4.703652783398866)
(8.3, 4.510859168781466)
(8.4, 4.2729945404414025)
(8.5, 3.9924355631174513)
(8.6, 3.6719854893705666)
(8.7, 3.314846150410917)
(8.8, 2.9245859644588084)
(8.9, 2.505104282289423)
(9.0, 2.060592426208783)
(9.1, 1.5954918117467607)
(9.2, 1.1144495705012383)
(9.3, 0.6222721175353085)
(9.4, 0.12387712726678882)
(9.5, -0.3757556023090465)
(9.6, -0.8716339061148982)
(9.7, -1.3588031320547123)
(9.8, -1.8323956462596418)
(9.9, -2.2876794688766067)
(10.0, -2.7201055544468487)};\addlegendentry{$y = 5 \sin(x)$}\addplot[draw=green, line width=2] coordinates {(-10.0, -10.0)
(-9.9, -9.9)
(-9.8, -9.8)
(-9.7, -9.7)
(-9.6, -9.6)
(-9.5, -9.5)
(-9.4, -9.4)
(-9.3, -9.3)
(-9.2, -9.2)
(-9.1, -9.1)
(-9.0, -9.0)
(-8.9, -8.9)
(-8.8, -8.8)
(-8.7, -8.7)
(-8.6, -8.6)
(-8.5, -8.5)
(-8.4, -8.4)
(-8.3, -8.3)
(-8.2, -8.2)
(-8.1, -8.1)
(-8.0, -8.0)
(-7.9, -7.9)
(-7.8, -7.8)
(-7.7, -7.7)
(-7.6, -7.6)
(-7.5, -7.5)
(-7.4, -7.4)
(-7.3, -7.3)
(-7.2, -7.2)
(-7.1, -7.1)
(-7.0, -7.0)
(-6.9, -6.9)
(-6.8, -6.8)
(-6.7, -6.7)
(-6.6, -6.6)
(-6.5, -6.5)
(-6.4, -6.4)
(-6.3, -6.3)
(-6.2, -6.2)
(-6.1, -6.1)
(-6.0, -6.0)
(-5.9, -5.9)
(-5.8, -5.8)
(-5.7, -5.7)
(-5.6, -5.6)
(-5.5, -5.5)
(-5.4, -5.4)
(-5.3, -5.3)
(-5.2, -5.2)
(-5.1, -5.1)
(-5.0, -5.0)
(-4.9, -4.9)
(-4.8, -4.8)
(-4.7, -4.7)
(-4.6, -4.6)
(-4.5, -4.5)
(-4.4, -4.4)
(-4.3, -4.3)
(-4.2, -4.2)
(-4.1, -4.1)
(-4.0, -4.0)
(-3.9, -3.9)
(-3.8, -3.8)
(-3.7, -3.7)
(-3.6, -3.6)
(-3.5, -3.5)
(-3.4, -3.4)
(-3.3, -3.3)
(-3.2, -3.2)
(-3.1, -3.1)
(-3.0, -3.0)
(-2.9, -2.9)
(-2.8, -2.8)
(-2.7, -2.7)
(-2.6, -2.6)
(-2.5, -2.5)
(-2.4, -2.4)
(-2.3, -2.3)
(-2.2, -2.2)
(-2.1, -2.1)
(-2.0, -2.0)
(-1.9, -1.9)
(-1.8, -1.8)
(-1.7, -1.7)
(-1.6, -1.6)
(-1.5, -1.5)
(-1.4, -1.4)
(-1.3, -1.3)
(-1.2, -1.2)
(-1.1, -1.1)
(-1.0, -1.0)
(-0.9, -0.9)
(-0.8, -0.8)
(-0.7, -0.7)
(-0.6, -0.6)
(-0.5, -0.5)
(-0.4, -0.4)
(-0.3, -0.3)
(-0.2, -0.2)
(-0.1, -0.1)
(0.0, 0.0)
(0.1, 0.1)
(0.2, 0.2)
(0.3, 0.3)
(0.4, 0.4)
(0.5, 0.5)
(0.6, 0.6)
(0.7, 0.7)
(0.8, 0.8)
(0.9, 0.9)
(1.0, 1.0)
(1.1, 1.1)
(1.2, 1.2)
(1.3, 1.3)
(1.4, 1.4)
(1.5, 1.5)
(1.6, 1.6)
(1.7, 1.7)
(1.8, 1.8)
(1.9, 1.9)
(2.0, 2.0)
(2.1, 2.1)
(2.2, 2.2)
(2.3, 2.3)
(2.4, 2.4)
(2.5, 2.5)
(2.6, 2.6)
(2.7, 2.7)
(2.8, 2.8)
(2.9, 2.9)
(3.0, 3.0)
(3.1, 3.1)
(3.2, 3.2)
(3.3, 3.3)
(3.4, 3.4)
(3.5, 3.5)
(3.6, 3.6)
(3.7, 3.7)
(3.8, 3.8)
(3.9, 3.9)
(4.0, 4.0)
(4.1, 4.1)
(4.2, 4.2)
(4.3, 4.3)
(4.4, 4.4)
(4.5, 4.5)
(4.6, 4.6)
(4.7, 4.7)
(4.8, 4.8)
(4.9, 4.9)
(5.0, 5.0)
(5.1, 5.1)
(5.2, 5.2)
(5.3, 5.3)
(5.4, 5.4)
(5.5, 5.5)
(5.6, 5.6)
(5.7, 5.7)
(5.8, 5.8)
(5.9, 5.9)
(6.0, 6.0)
(6.1, 6.1)
(6.2, 6.2)
(6.3, 6.3)
(6.4, 6.4)
(6.5, 6.5)
(6.6, 6.6)
(6.7, 6.7)
(6.8, 6.8)
(6.9, 6.9)
(7.0, 7.0)
(7.1, 7.1)
(7.2, 7.2)
(7.3, 7.3)
(7.4, 7.4)
(7.5, 7.5)
(7.6, 7.6)
(7.7, 7.7)
(7.8, 7.8)
(7.9, 7.9)
(8.0, 8.0)
(8.1, 8.1)
(8.2, 8.2)
(8.3, 8.3)
(8.4, 8.4)
(8.5, 8.5)
(8.6, 8.6)
(8.7, 8.7)
(8.8, 8.8)
(8.9, 8.9)
(9.0, 9.0)
(9.1, 9.1)
(9.2, 9.2)
(9.3, 9.3)
(9.4, 9.4)
(9.5, 9.5)
(9.6, 9.6)
(9.7, 9.7)
(9.8, 9.8)
(9.9, 9.9)
(10.0, 10.0)};\addlegendentry{$y = x$}
\end{axis}\end{tikzpicture}\end{center}
